examples/csharp/Dockerfile

```diff
@@ -24,9 +23,11 @@ COPY src/Twilio/Rest src/Twilio/Rest
 RUN rm -rf test/Twilio.Test/*/ test/Twilio.Test/*Cluster*
 COPY unit-test/Rest integrationTest/Rest test/Twilio.Test/
 
-RUN dotnet restore
-RUN	dotnet build Twilio.sln
-CMD ["/bin/bash", "-c", "set -o pipefail && sleep 10 && dotnet test test/Twilio.Test/Twilio.Test.csproj --no-build 2>&1 | tee /local/test-report.out"]
+RUN make install
+
+COPY test.sh .
+ARG SONAR_TOKEN
+ENV SONAR_TOKEN=$SONAR_TOKEN
 
-RUN apt-get -y install build-essential
-CMD if [ "$SONAR_TOKEN" = "" ] ; then echo Skipping sonar analysis ;  else make cover PROJECT_NAME=twilio_twilio-oai-generator-csharp SONAR_SOURCES="/d:sonar.inclusions=src/Twilio/Rest/**/* /d:sonar.cs.analyzeGeneratedCode=true /d:sonar.exclusions=test/Twilio.Test/**/*" ; fi
+# pipefail prevents errors in a pipeline from being masked.
+CMD ["/bin/bash", "-c", "set -o pipefail && chmod u+r+x /app/twilio-csharp/test.sh && /app/twilio-csharp/test.sh 2>&1 | tee /local/test-report.out"]
```

Comment:
Should remove the `chmod` from here and just do it on the file itself.

Reply:
Removing chmod causes the tests to fail with error bin/bash: line 1: /app/twilio-csharp/test.sh: Permission denied